src/features/nft/bidBuyModal/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
```

Comment:
Change the file name.

Reply:
@dbizhev I follow the title of the design for filename. It it is wrong, please suggest the filename

Reply:
@muhadovic rename it to BidOfferModal.tsx

Reply:
@dbizhev , it is wrong. 

https://github.com/pastelnetwork/pastel-electron-wallet/pull/105
Please check 
![image](https://user-images.githubusercontent.com/49214944/120387239-ebad2300-c329-11eb-84a6-b2329b724535.png)